command-expansion-server/src/app.ts

```diff
@@ -353,6 +364,117 @@ app.post('/expand-all-activity-instances', async (req, res, next) => {
   return next();
 });
 
+export interface SeqBuilder {
```

Comment:
:large_blue_circle: Is this commit both fixing simulated activity batch loaders updated in the last commit and adding the SeqBuilder capability? I want to make sure I'm understanding the ideas/intentions introduced in this commit given the commit message.

Reply:
Definitely a rebase error. I think all of this should be in 8128bba6d51d9e9b362a5b649c8bc99fefd241d8.